sim/shaman/shocks.go

```diff
@@ -95,6 +83,8 @@ func (shaman *Shaman) registerFlameShockSpell(shockTimer *core.Timer) {
 		TickLength:          time.Second * 3,
 		AffectedByCastSpeed: true,
 
+		//TODO: Snapshot?
+		//TODO: I don't know what 834/6 is
```

Comment:
This was the 834 base damage of [Flame Shock Rank 9](https://www.wowhead.com/wotlk/spell=49233/flame-shock) divided by 6 ticks of the DoT